camera_calibration/src/camera_calibration/calibrator.py

```diff
@@ -246,6 +246,11 @@ def _get_circles(img, board, pattern):
         (ok, corners) = cv2.findCirclesGrid(
             mono_arr, (board.n_rows, board.n_cols), flags=flag, blobDetector=detector)
 
+        if ok:
```

Comment:
[nits] add comments about your intention for this implementation.

Reply:
done